src/app/core/rpc/rpx.service.ts

```diff
@@ -23,7 +24,36 @@ export class RPXService {
   private listenerCount: number = 0;
   listeners: { [id: string]: boolean } = {};
 
-  constructor(public zone: NgZone) { }
+  constructor(
+    public zone: NgZone,
+  ) {
+
+    window.ipc.on('rx-ipc-check-listener', (event, channel) => {
+      const replyChannel = 'rx-ipc-check-reply:' + channel;
+      if (this.listeners[channel]) {
+        event.sender.send(replyChannel, true);
+      } else {
+        event.sender.send(replyChannel, false);
+      }
+    });
+
+    window.ipc.on('front-choosewallet', (method, wallets) => {
```

Comment:
TODO: comment out or move to service.

Reply:
I'm seeing lots of commenting out of multiwallet code so I presume this must be as well